tests/Unit/Models/GroupTest.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+namespace Tests\Unit\Models;
+
+use App\Models\Bookmark;
+use App\Models\Group;
+use App\Models\Group\Member;
+use App\Models\Group\Workspace;
+use App\Models\LegiScan\Bill;
+use App\Models\User;
+use Illuminate\Support\Facades\DB;
+use Tests\TestCase;
+
+class GroupTest extends TestCase
+{
+    /** @test */
+    public function groupsHaveMembers()
+    {
+        $group = Group::factory()->create();
+        $user1 = User::factory()->create();
+        $user2 = User::factory()->create();
+        User::factory()->count(4)->create();
+        Member::factory()->create([
+            'group_id' => $group->id,
+            'user_id'  => $user1->id,
+        ]);
+        Member::factory()->create([
+            'group_id' => $group->id,
+            'user_id'  => $user2->id,
+        ]);
+
+        $this->assertEquals(2, $group->members->count());
+        $this->assertEquals(
+            [$user1->id, $user2->id],
+            $group->members->pluck('id')->sort()->toArray(),
+        );
+    }
+
+    /** @test */
+    public function groupsDontHaveMembersWhenNoneAdded()
+    {
+        $group = Group::factory()->create();
+
+        $this->assertEquals(0, $group->members->count());
+    }
+
+    /** @test */
+    public function groupsCanBeScopedToUsers()
+    {
+        $groups = Group::factory()->count(5)->create();
+        $user1  = User::factory()->create();
+        $user2  = User::factory()->create();
+        User::factory()->count(5)->create();
```

Comment:
PLEASE add whitespace to things. It's so hard to read when everything is just smashed together.

Reply:
What does having 5 Users prove that having 2 Users doesn't?

Reply:
Fixed!